Clarify that podDisruptionBudget=0 is set so that Kube does not interfere with Drain Cleaner (#6616)
strimzi/strimzi-kafka-operator: documentation/modules/managing/proc-drain-cleaner-deploying.adoc

----
+
Reducing the maximum pod disruption budget to zero prevents Kubernetes from automatically evicting the pods in case of voluntary disruptions,

> leaving the floor for pods to be evicted by the Strimzi Drain Cleaner without conflict.

Reviewer: ```suggestion
leaving the Strimzi Drain Cleaner free to evict pods without conflict.
```

Reviewer 2: It is the joint work of Drain Cleaner, Cluster Operator and K8s Scheduler to actually move pods away from the cordoned node.

Reviewer 3: Yeah, I think Federico has a point. Maybe something like

```suggestion
leaving the Strimzi Drain Cleaner and Strimzi Cluster Operator to roll the pod which will be scheduled by Kubernetes on a different worker node.
```
